### DISCUSSION TITLE
I can't login with the quickstart?

### DISCUSSION THREAD

**Discussion by PythonAI1234** [ig-python/ig-markets-api-python-library#324] [Q&A] [answered]
I changed the config file just like the Quickstart teaches 
and typed the following code

from trading_ig.rest import IGService
from trading_ig.config import config
ig_service = IGService(config.username, config.password, config.api_key)
ig = ig_service.create_session()
ig

Yet it says
can't import config from config file

Exception: Environment variable 'IG_SERVICE_USERNAME' doesn't exist

Why is this so??
I entered my information into the config file with a notepad and renamed it.
Thanks in advance.

**Comment by breitdom** [comment]
Ty your login data here: https://labs.ig.com/sample-apps/api-companion/index.html

Are you using a DEMA account? If so, please note the comments.

**Reply by PythonAI1234** [reply]
I logged into the api companion alright.
Is the accNumber in the trading_ig_config.py the same as the "accountId" in the account details in the API companion?
I don't think I have made any mistake entering into trading_ig_config.py.

**Reply by PythonAI1234** [reply]
I am using a LIVE account

**Reply by breitdom** [reply]
Everything was still working yesterday. I have now tried again and can no longer access my account via the Python API or Companion.

Error: trading_ig.rest.IGException: HTTP error: 401 {"errorCode": "error.security.client-suspended"}

I can't tell right now if IG API has a problem in general.

**Reply by PythonAI1234** [reply]
I changed the thing in the folder under windows username 
C:/user/my_name/trading-ig/trading_ig_config.py
If this is wrong, correct me.
This is where I found the template so I assume this must be the correct folder.

**Reply by breitdom** [reply]
```
`from trading_ig.config import config

print(config.username)
print(config.password)
print(config.api_key)
print(config.acc_type)
print(config.acc_number)`
```

try something like this for validation...

**Reply by PythonAI1234** [reply]
it just says this
Not very useful
maybe I'm not supposed to use anaconda and jupyter notebook I dunno
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
~\anaconda3\lib\site-packages\trading_ig\config.py in __getattr__(self, key)
     26         try:
---> 27             return os.environ[env_var]
     28         except KeyError:

~\anaconda3\lib\os.py in __getitem__(self, key)
    674             # raise KeyError with the original key value
--> 675             raise KeyError(key) from None
    676         return self.decodevalue(value)

KeyError: 'IG_SERVICE_USERNAME'

During handling of the above exception, another exception occurred:

Exception                                 Traceback (most recent call last)
<ipython-input-3-97608f9a8987> in <module>
      1 from trading_ig.config import config
      2 
----> 3 print(config.username)
      4 print(config.password)
      5 print(config.api_key)

~\anaconda3\lib\site-packages\trading_ig\config.py in __getattr__(self, key)
     27             return os.environ[env_var]
     28         except KeyError:
---> 29             raise Exception("Environment variable '%s' doesn't exist" % env_var)
     30 
     31 

Exception: Environment variable 'IG_SERVICE_USERNAME' doesn't exist

**Reply by breitdom** [reply]
Yes, your python environment is not installed / cunfigured correctly.

"Environment variable '%s' doesn't exist" % env_var

**Reply by PythonAI1234** [reply]
I have no idea what to do now as a noob. I don't know anything about python environments and almost always load up jupyter notebook on anaconda lol.

**Comment by PythonAI1234** [comment, marked as answer]
for documentation sake
I solved the thing.
The trading_ig_config.py had to be put under the python directory?? or the directory of .py I am running with.(I dunno which one) In my case the /Users/username. 

Then I encountered the error no module named Crypto, I uninstall crypto then reinstalled pycryptodome to no avail.
Then I manually changed the folder under Lib/site-packages from 'crypto' to 'Crypto' manually and all good
